website/docs/core-concepts/components/remote-state.md

```diff
@@ -26,7 +26,8 @@ In Atmos, Terraform Remote State is implemented by using these modules:
 modules to work with [Atmos CLI config](/cli/configuration) and [Atmos stacks](/core-concepts/stacks). The provider processes stack
 configurations to get the final config for an Atmos component in an Atmos stack. The final component config is then used by
 the [remote-state](https://github.com/cloudposse/terraform-yaml-stack-config/tree/main/modules/remote-state) Terraform module to return the remote
-state for the component in the stack.
+state for the component 
+stack.
```

Comment:
why is this moved to two lines?

Reply:
Random bad keystroke 😬 Thanks for catching, I'll correct.